lib/entities/vendor.dart

```diff
@@ -0,0 +1,71 @@
+// Class used to represent a vendor.
+class Vendor implements Comparable<Vendor> {
+  // Vendor ID.
+  String? id;
```

Comment:
Same, if possible it should not be nullable

Reply:
we'll keep this as nullable for now considering null/nil values can come from native SDKs.